sdk/storage/storage-blob/src/ServiceURL.ts

```diff
@@ -62,6 +64,20 @@ export class ServiceURL extends StorageURL {
     this.serviceContext = new Service(this.storageClientContext);
   }
 
+  static fromConnectionString(
+    connectionString: string,
+    pipelineOptions?: INewPipelineOptions
```

Comment:
`INewPipelineOptions`  would change into `BobConnectionOptions`, is it ?

Reply:
that was the plan: INewPipelineOptions would be moved inside BlobConnectionOptions.